### DISCUSSION TITLE
Extend Shares Feature (CRUD Support)

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#14949] [Feature Requests]
**While directus already has a great shares feature to create views for non-authenticated users, I'd love to see a create (or even full CRUD) support. This would allow users to request data from non-users and also create single use forms.**

The setup of a new share could be identical to the current one as the permissions for the operations would be managed by the  selected role-permission.

Related to [2977](https://github.com/directus/directus/discussions/2977)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
